### PR TITLE
EPS-477: PHPCS Check for plugin

### PR DESCRIPTION
**Pull request by vijetaR**

### CODE REVIEW

File: timeline-for-beaver-builder.php [hunk 1]
```diff
@@ -27,15 +27,15 @@ class BSFBBTimeline {
 		/**
 		 * To initializ new object
 		 */
-		function __construct() {
+		public function __construct() {
```
Comment (sushmak02) [inline timeline-for-beaver-builder.php:30]: For above errors add above class name related errors in phpcs ignore list

File: timeline-for-beaver-builder-module/includes/frontend.php [hunk 2]
```diff
@@ -143,17 +145,17 @@
...
+					<<?php echo esc_html( $settings->tmtitle_tag ); ?> class="bb-timline-title bb-tm-title-<?php echo esc_html( $timeline1->timeline_title_align ); ?>">
+						<?php echo esc_html( $timeline1->timeline_title ); ?>
+					</<?php echo esc_html( $settings->tmtitle_tag ); ?>>
 					<!--/.Timline-Title-->
 					<div class="tm-title-border-bottom">
 						<span class="bb-tmlabel-border-bottom"></span>
 					</div>
 					<!--Timline-description-->
-					<div class="bb-timline-dec"><?php echo $timeline1->timeline_editor; ?></div>
+					<div class="bb-timline-dec"><?php echo wp_kses_post( $timeline1->timeline_editor ); ?></div>
```
Comment (sushmak02) [inline timeline-for-beaver-builder-module/includes/frontend.php:158]: Make sure these changes are tested 100%

**Review by sushmak02 — APPROVED** [review]

File: timeline-for-beaver-builder.php (hunk 1 above)
Comment (vijetaR) [reply timeline-for-beaver-builder.php:30]: I have ignored it in ruleset for some, but somehow it's failing for other checks

File: timeline-for-beaver-builder-module/includes/frontend.php [hunk 3]
```diff
@@ -77,12 +78,12 @@
 					</div>
 
 					<!--Timline-description-->
-					<div class="bb-timline-dec"><?php echo $timeline1->timeline_editor; ?></div>
+					<div class="bb-timline-dec"><?php echo wp_kses_post( $timeline1->timeline_editor ); ?></div>
```
Comment (sushmak02) [inline timeline-for-beaver-builder-module/includes/frontend.php:81]: Make sure this is tested

Comment (sushmak02) [reply timeline-for-beaver-builder-module/includes/frontend.php:81]: If we are providing editor which allows HTML  too it should work properly, backward comp needs to be tested

**Review by sushmak02 — APPROVED** [review]

File: timeline-for-beaver-builder-module/includes/frontend.php (hunk 3 above)
Comment (vijetaR) [reply timeline-for-beaver-builder-module/includes/frontend.php:81]: Tested. Working fine with HTML tags

**Review by sushmak02 — APPROVED** [review]

**Pull request merged**